services/debugger/destination/eventDeliveryStatusUploader.go

```diff
@@ -63,7 +63,7 @@ func NewHandle(backendConfig backendconfig.BackendConfig, opts ...Opt) (Destinat
 		log:              logger.NewLogger().Child("debugger").Child("destination"),
 	}
 	var err error
-	url := fmt.Sprintf("%s/dataplane/v2/eventUploads", h.configBackendURL)
+	url := fmt.Sprintf("%s/dataplane/v2/eventDeliveryStatus", h.configBackendURL)
```

Comment:
this was wrong?!

Reply:
Yep

Reply:
so previously we didn't upload event delivery statuses? Or both urls work?

Reply:
This went in my Badger PR.... so it is not on any customer but is present in master

Reply:
Both URL's don't work so this is a bug which was caught by QA

Reply:
It is available.
As part of a previous PR, there was some mixup while copy pasting.
This just corrects it.

Reply:
also yes.
`/dataplane/v2/eventDeliveryStatus` is the correct endpoint here